### DISCUSSION TITLE
Frigate .15 beta 1 Issues with clip

### DISCUSSION THREAD

**Discussion by kiloptero** [blakeblackshear/frigate#14726] [Ask A Question]
![IMG_2892](https://github.com/user-attachments/assets/2f188b69-8aa6-4f82-8801-753adea8c9c3)
Due the last update  when I send a clip with frigate notification the clip doesn't show .. in other part , the snapshot and image works perfect. Already change the config in record

record:
  sync_recordings: True
  enabled: true
  retain:
    days: 10
    mode: all
  alerts:
    retain:
      days: 10
      mode: all
  detections:
    retain:
      days: 10
      mode: all
snapshots:
  enabled: true
  retain:
    default: 500

![IMG_2895](https://github.com/user-attachments/assets/20e00ea9-f02c-45d2-98dc-f397725cf043)

**Comment by NickM-27** [comment]
are there any frigate logs?

**Comment by kiloptero** [comment]
[frigate-logs.txt](https://github.com/user-attachments/files/17604208/frigate-logs.txt)

**Reply by NickM-27** [reply]
your recording stream is not working, check go2rtc logs

**Reply by kiloptero** [reply]
Weird.. I can see every recording with issues. :(

**Comment by kiloptero** [comment]
[go2rtc frigate .14.txt](https://github.com/user-attachments/files/17604389/go2rtc.frigate.14.txt)
[go2rtc.txt](https://github.com/user-attachments/files/17604390/go2rtc.txt)

both go2rtc logs.. I can't see any reference to the clips:(
